### PR TITLE
Clickhouse cdc data types

### PR DESCRIPTION
**Pull request by pankaj-peerdb**

### CODE REVIEW

**Review by serprex — APPROVED** [review]

File: flow/connectors/clickhouse/normalize.go [hunk 1]
```diff
@@ -149,19 +149,16 @@ func (c *ClickhouseConnector) NormalizeRecords(req *model.NormalizeRecordsReques
...
+			clickhouseType, err := qValueKindToClickhouseType(colType)
+			if err != nil {
+				return nil, fmt.Errorf("error while converting column type to clickhouse type: %w", err)
 			}
-			projection.WriteString(fmt.Sprintf("%s(_peerdb_data, '%s') AS %s, ", extractionFuction, cn, cn))
+			if clickhouseType == "DateTime64(6)" {
+				clickhouseType = "String"
+			}
+
+			projection.WriteString(fmt.Sprintf("JSONExtract(_peerdb_data, '%s', '%s') AS %s, ", cn, clickhouseType, cn))
```
Comment (serprex) [inline flow/connectors/clickhouse/normalize.go:161]: quote `cn`

Comment (pankaj-peerdb) [reply flow/connectors/clickhouse/normalize.go:161]: Is it just a hygiene or any specific reason for this. Column names cannot have space or anything right

Comment (serprex) [reply flow/connectors/clickhouse/normalize.go:161]: [Yes, they can.](https://www.postgresql.org/docs/7.0/syntax525.htm#:~:text=Names%20containing%20other%20characters%20may,always%20folded%20to%20lower%20case.) Our codebase has generally relied on customers keeping plain names, at least without internal quotes, but this is something we need to fix

Comment (serprex) [reply flow/connectors/clickhouse/normalize.go]: clickhouse identifier syntax: https://clickhouse.com/docs/en/sql-reference/syntax#identifiers

**Pull request merged**